### ISSUE TITLE
README: Mention difference between this and other forwarders

### ISSUE THREAD

**Issue by rfay** [cmuench/inotify-proxy#8]
Could you mention in the README the difference between this and https://github.com/mhallin/notify-forwarder and https://github.com/TomFrost/fs_eventbridge? 

I'm surprised because I think those two have a host component, and it seems like this one doesn't.

**Comment by theblop** [comment]
Looking at the code, it seems this is not a forwarder nor a proxy, but is supposed to run inside a VM or container and it just periodically walks the watched directories and generates inotify events...

**Comment by cmuench** [comment]
@theblop right. It's a revamped version of the https://www.npmjs.com/package/inotify-proxy which in the background compiles a C program.
